pkgs/development/python-modules/git-annex-adapter/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, isPy3k, fetchPypi, pygit2 }:
+
+buildPythonPackage rec {
+  name = "git-annex-adapter-${version}";
+  version = "0.2.0";
+  format = "wheel";
```

Comment:
Building from the actual source instead of a wheel has preference. You can download a tarball from https://github.com/alpernebbi/git-annex-adapter

Reply:
Could we mention this in the manual?

Reply:
We may add it explicitly with regards to wheels, but its pretty much the rule throughout Nixpkgs that we prefer to build from source.

Reply:
Building right from GitHub now :)